### DISCUSSION TITLE
Version 1.2.0 and wallet problem

### DISCUSSION THREAD

**Discussion by pele2** [Chia-Network/chia-blockchain#7762] [General]
After upgrading to version 1.2.0 (sync is full) I have a problem with the wallet, I had 4 chia, and now have added transactions:

Incoming
xch1vjn4u7e6gwzw47h7mdc77zq7s2qdssw4jl8hum3valwf72evxf9qz89535
07/09/2021 9:16:27
Confirmed at height 546326
1,749999999999
0
Outgoing
xch1fuveejkask3jwv5nmzygwqcdr86rtlkuqwullzmttqw55rp6fw5qjez88n
07/09/2021 7:34:21
Confirmed at height 545951
0,000000000001
0
 
but the resulting balance I see is 3.999999999998 XCH. I am attaching the screen, where is the problem?
The transactions earlier are ok.

Thank you for the explanation.

![opfokncmlngnhkna](https://user-images.githubusercontent.com/87894474/126862689-b6cc8d9d-aec2-47ce-988e-7645a0d87e9a.png)
![gnlgacanfihpigpa](https://user-images.githubusercontent.com/87894474/126862688-f58d46f0-5653-4881-905b-4d7c2484f091.png)

**Comment by SlugFiller** [comment]
Did you create a pool NFT? Creating an NFT requires burning one mojo, or 0.000000000001 XCH. This would explain the outgoing transaction, as well as your balance. For the second incoming address, my best guess is a change address, for spending 1 mojo out of a 1.75 XCH address. But it wouldn't explain why they're at such different heights.

**Reply by pele2** [reply]
Yes, that will be it, I created one NFT plot as a test, but I don't understand what the incoming one means and why it is not in the total wallet

**Reply by keliew** [reply]
@pele2 https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#what-is-the-coin-model
